cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -379,7 +379,7 @@ func (command *Evaluate) Execute(args []string) (err error) {
 	}
 
 	_ = assessmentsPerModel.WalkByScore(func(model model.Model, assessment metrics.Assessments, score uint64) (err error) {
-		command.logger.Printf("Evaluation score for %q (%q): %s", model.ID(), assessment.Category(totalScore).ID, assessment)
+		command.logger.Printf("Evaluation score for %q (%q): cost=%f, %s", model.ID(), assessment.Category(totalScore).ID, model.Cost(), assessment)
```

Comment:
lets make this `%.2f` to have two decimal places at all times. Makes no sense having `0.001` dollars there. The full precision value will always be in the CSV anyways.